adal/devApps/XFormsApp.iOS/AppDelegate.cs

```diff
@@ -52,6 +52,9 @@ public partial class AppDelegate : global::Xamarin.Forms.Platform.iOS.FormsAppli
         public override bool FinishedLaunching(UIApplication app, NSDictionary options)
         {
             global::Xamarin.Forms.Forms.Init();
+            #if ENABLE_TEST_CLOUD
+            Xamarin.Calabash.Start();
```

Comment:
>Calabash [](start = 20, length = 8)

I read on app center tonight that they are deprecating Calabash [https://docs.microsoft.com/en-us/appcenter/test-cloud/preparing-for-upload/calabash](https://docs.microsoft.com/en-us/appcenter/test-cloud/preparing-for-upload/calabash) ...should we use something else?

Reply:
nevermind...i see they are still supporting it through Xamarin.UITest, which we are using. seems weird. lol.

---
In reply to: [235236921](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1418#discussion_r235236921) [](ancestors = 235236921)

Reply:
Yeah. I saw the same. scared me for a second. lol